lib/ansible/modules/cloud/cloudstack/cs_template.py

```diff
@@ -390,6 +401,8 @@
 
 # import cloudstack common
 from ansible.module_utils.cloudstack import *
+import requests
```

Comment:
please make a comment here that is okay here to depend on requests since this module depends on cs which depends on requests.